keda/templates/deployment.yaml

```diff
@@ -41,30 +49,48 @@ spec:
           - name: grpc-certs
             mountPath: /grpccerts
           {{- end }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
         - name: {{ .Values.operatorName }}-metrics-apiserver
-          image: {{ .Values.image.metricsAdapter }}
+          image: "{{ .Values.image.metricsAdapter }}:{{ .Chart.AppVersion }}"
```

Comment:
Ok for you if we add this @ahmelsayed? It's fine for me.

Reply:
I'd prefer if there is a way to override the full image name because it enables using helm for deploying a custom keda. [like this](https://github.com/kedacore/keda#deploying-custom-keda)